include/arch/arm/aarch64/scripts/linker.ld

```diff
@@ -148,7 +151,6 @@ SECTIONS
 
     SECTION_PROLOGUE(_TEXT_SECTION_NAME_2,,)
```

Comment:
The recent Cortex-M `_TEXT_SECTION_NAME_2` clean-up could be done here too? (or possibly in a separate PR)

Reply:
Done, see #22673.